common/configuration.adoc

```diff
@@ -104,7 +102,7 @@ endif::ssh[]
 . Enter the `appliance_console` command. The {product-title} appliance summary screen displays.
 . Press *Enter* to manually configure settings.
 . Select *5) Configure Database* from the menu.
-. You are prompted to create or fetch a security key. Since this is not the first {product-title} appliance, choose *2) Fetch key from remote machine*. For worker and multi-region setups, use this option to copy key from another appliance. 
+. You are prompted to create or fetch a security key. Since this is not the first {product-title} appliance, choose *2) Fetch key from remote machine*. For worker and multi-region setups, use this option to copy key from another appliance.
```

Comment:
Instead of "copy key" in the last phrase, I would write out the full "copy the security key" since its not echoing UI language.